apps/emails/templates/nrh-default-contribution-confirmation-email.html

```diff
@@ -4,329 +4,250 @@
-                  <div style="width: 100% !important;">
-                    <!--[if (!mso)&(!IE)]><!-->
-                    <div
-                      style="padding: 0px;border-top: 0px solid transparent;border-left: 0px solid transparent;border-right: 0px solid transparent;border-bottom: 0px solid transparent;">
-                      <!--<![endif]-->
-
-                      <table style="font-family:'Lato',sans-serif;" role="presentation" cellpadding="0" cellspacing="0"
-                        width="100%" border="0">
-                        <tbody>
-                          <tr>
-                            <td
-                              style="overflow-wrap:break-word;word-break:break-word;padding:10px;font-family:'Lato',sans-serif;"
-                              align="left">
-
-                              <div>
-                                <div style="background-color: #e0e0e0; padding: 24px">
-                                  <strong>Contribution Details:</strong>
-                                  <p>Date Received: {{ contribution_date }}</p>
-                                  <p>Email: {{ contributor_email }}</p>
-                                  {% if contribution_interval == "one_time" %}
-                                  <p>Amount Contributed: {{ contribution_amount }}</p>
-                                  {% else %}
-                                  <p>Amount Contributed: {{ contribution_amount}}/{{contribution_interval_display_value}}</p>
-                                  {% endif %}
-                                </div>
-                              </div>
-
-                            </td>
-                          </tr>
-                        </tbody>
-                      </table>
-
-
-                      <!--[if (!mso)&(!IE)]><!-->
-                    </div>
-                    <!--<![endif]-->
-                  </div>
-                </div>
-                <!--[if (mso)|(IE)]></td><![endif]-->
-                <!--[if (mso)|(IE)]></tr></table></td></tr></table><![endif]-->
-              </div>
-            </div>
+          <div class="u-row-container">
+            <table width="100%" cellpadding="0" cellspacing="0" border="0">
+              <tr>
+                <td style="padding-right: 0px;padding-left: 0px;" align="left">
+                  <p class="disclaimer inner-center">
+                    No goods or services were provided in exchange for this contribution. This receipt may be used for tax purposes.
+                    {{ org_name }} is a {{ company_type }}
+                    {% if for_profit == true %}
+                    for profit
+                    {% else %}
+                    nonprofit
+                    {% endif %}
+                    organization
+                    {% if tax_id != "" %}
+                    with a Federal Tax ID #{{ tax_id }}
+                    {% endif %}.
+                  </p>
```

Comment:
Didn't catch this the first time through, but looking at [this comment in the JIRA ticket,](https://news-revenue-hub.atlassian.net/browse/DEV-2783?focusedCommentId=61976) this part of the template needs to be changed along these lines:

```suggestion
                  <p class="disclaimer inner-center">
                    No goods or services were provided in exchange for this contribution.&nbsp;
                    {% if non_profit %}
                    This receipt may be used for tax purposes.&nbsp;
                    {{ rp_name }} is a 501(c)(3) nonprofit organization {% if tax_id %}with a Federal Tax ID
                    #{{tax_id}}{% endif %}.
                    {% else %}
                    Contributions to {{ rp_name }} are not deductible as charitable donations.
                    {% endif %}
                  </p>
```

Note that I've renamed `org_name` to `rp_name` because that's what is called for in ticket. 

I've also renamed `for_profit` to `non_profit`, which is the original name on the revenue program model. This should be a little more straightforward than negating the model field name.

Reply:
Didn't see that comment when working in this ticket. Thanks for pointing that out.